src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -852,6 +869,16 @@ private[snowflake] object EmptySnowflakeSQLStatement {
   def apply(): SnowflakeSQLStatement = new SnowflakeSQLStatement()
 }
 
+private[snowflake] object ConstantStringVal {
+  def apply(l: Any): StatementElement = {
+    if (l == null || l.toString == "null") {
```

Comment:
Do we need to consider " null", "Null", etc?

Reply:
Good point. I am simply porting over old behavior, but we should do that. Will make the change.